packages/css-library/config.js

```diff
@@ -1,5 +1,7 @@
 const { transform } = require('@divriots/style-dictionary-to-figma');
 
+const filtersUswdsTokens = token => !token.name.includes('uswds');
```

Comment:
filter is passed to formats. I tried doing this at the source level with a glob but ran into a build error, I think because of how we're referencing primitives as values.